apluslms_roman/configuration.py

```diff
@@ -1,19 +1,27 @@
+from itertools import chain
 from os import listdir
-from os.path import join, isdir, isfile
+from os.path import basename, join, isdir, isfile, relpath
 
 from apluslms_yamlidator.document import Document
 from apluslms_yamlidator.utils.collections import OrderedDict
 from apluslms_yamlidator.utils.version import Version
 
 from .utils.translation import _
 
-DEFAULT_NAME = 'course'
+DEFAULT_NAME = 'roman'
+DEFAULT_NAMES = ('roman', 'course')
 DEFAULT_PREFIXES = ('yml', 'yaml', 'json')
```

Comment:
We could define `DEFAULT_FILENAME = "%s.%s" % (DEFAULT_NAMES[0], DEFAULT_PREFIXES[0])` here.